data/processed/hesa/total_stem_postgraduates.csv

```diff
@@ -0,0 +1,201 @@
+2014,UKK1,2018,1750.0
+2015,UKK1,2018,1755.0
+2016,UKK1,2018,1885.0
+2017,UKK1,2018,1925.0
+2018,UKK1,2018,1950.0
+2014,UKD3,2018,2330.0
+2015,UKD3,2018,2345.0
+2016,UKD3,2018,2385.0
+2017,UKD3,2018,2600.0
+2018,UKD3,2018,2710.0
+2014,UKJ3,2018,1625.0
+2015,UKJ3,2018,1600.0
+2016,UKJ3,2018,1560.0
+2017,UKJ3,2018,1520.0
+2018,UKJ3,2018,1480.0
+2014,UKG1,2018,0.0
+2015,UKG1,2018,15.0
+2016,UKG1,2018,20.0
+2017,UKG1,2018,20.0
+2018,UKG1,2018,25.0
+2014,UKM6,2018,10.0
+2015,UKM6,2018,20.0
+2016,UKM6,2018,30.0
+2017,UKM6,2018,60.0
+2018,UKM6,2018,55.0
+2014,UKI4,2018,1105.0
+2015,UKI4,2018,1070.0
+2016,UKI4,2018,1050.0
+2017,UKI4,2018,1010.0
+2018,UKI4,2018,1025.0
+2014,UKI3,2018,5205.0
+2015,UKI3,2018,5425.0
+2016,UKI3,2018,5730.0
+2017,UKI3,2018,5825.0
+2018,UKI3,2018,6190.0
+2014,UKJ4,2018,530.0
+2015,UKJ4,2018,530.0
+2016,UKJ4,2018,550.0
+2017,UKJ4,2018,530.0
+2018,UKJ4,2018,560.0
+2014,UKD4,2018,655.0
+2015,UKD4,2018,620.0
+2016,UKD4,2018,620.0
+2017,UKD4,2018,690.0
+2018,UKD4,2018,695.0
+2014,UKF2,2018,1260.0
+2015,UKF2,2018,1280.0
+2016,UKF2,2018,1195.0
+2017,UKF2,2018,1100.0
+2018,UKF2,2018,1170.0
+2014,UKF3,2018,115.0
+2015,UKF3,2018,140.0
+2016,UKF3,2018,155.0
+2017,UKF3,2018,150.0
+2018,UKF3,2018,160.0
+2014,UKD7,2018,1010.0
+2015,UKD7,2018,970.0
+2016,UKD7,2018,1065.0
+2017,UKD7,2018,1155.0
+2018,UKD7,2018,1210.0
+2014,UKM5,2018,445.0
+2015,UKM5,2018,470.0
+2016,UKM5,2018,480.0
+2017,UKM5,2018,500.0
+2018,UKM5,2018,460.0
+2014,UKE2,2018,670.0
+2015,UKE2,2018,670.0
+2016,UKE2,2018,655.0
+2017,UKE2,2018,650.0
+2018,UKE2,2018,675.0
+2014,UKN0,2018,830.0
+2015,UKN0,2018,850.0
+2016,UKN0,2018,830.0
+2017,UKN0,2018,915.0
+2018,UKN0,2018,875.0
+2014,UKC2,2018,1010.0
+2015,UKC2,2018,1045.0
+2016,UKC2,2018,1040.0
+2017,UKC2,2018,1015.0
+2018,UKC2,2018,1030.0
+2014,UKI5,2018,135.0
+2015,UKI5,2018,150.0
+2016,UKI5,2018,130.0
+2017,UKI5,2018,125.0
+2018,UKI5,2018,110.0
+2014,UKI6,2018,95.0
+2015,UKI6,2018,100.0
+2016,UKI6,2018,100.0
+2017,UKI6,2018,75.0
+2018,UKI6,2018,70.0
+2014,UKI7,2018,615.0
+2015,UKI7,2018,545.0
+2016,UKI7,2018,470.0
+2017,UKI7,2018,465.0
+2018,UKI7,2018,530.0
+2014,UKG2,2018,200.0
+2015,UKG2,2018,185.0
+2016,UKG2,2018,210.0
+2017,UKG2,2018,160.0
+2018,UKG2,2018,155.0
+2014,UKE3,2018,1530.0
+2015,UKE3,2018,1520.0
+2016,UKE3,2018,1460.0
+2017,UKE3,2018,1425.0
+2018,UKE3,2018,1410.0
+2014,UKJ2,2018,1385.0
+2015,UKJ2,2018,1370.0
+2016,UKJ2,2018,1295.0
+2017,UKJ2,2018,1220.0
+2018,UKJ2,2018,1240.0
+2014,UKC1,2018,575.0
+2015,UKC1,2018,570.0
+2016,UKC1,2018,595.0
+2017,UKC1,2018,640.0
+2018,UKC1,2018,650.0
+2014,UKM8,2018,1920.0
+2015,UKM8,2018,1990.0
+2016,UKM8,2018,2120.0
+2017,UKM8,2018,2145.0
+2018,UKM8,2018,2195.0
+2014,UKG3,2018,2415.0
+2015,UKG3,2018,2430.0
+2016,UKG3,2018,2455.0
+2017,UKG3,2018,2525.0
+2018,UKG3,2018,2660.0
+2014,UKL1,2018,800.0
+2015,UKL1,2018,825.0
+2016,UKL1,2018,930.0
+2017,UKL1,2018,920.0
+2018,UKL1,2018,1010.0
+2014,UKE4,2018,1585.0
+2015,UKE4,2018,1615.0
+2016,UKE4,2018,1600.0
+2017,UKE4,2018,1660.0
```

Comment:
In the schema the value is marked as an `int` but values are floats

Reply:
As above